control/injector/Injector.php

```diff
@@ -594,6 +594,20 @@ public function inject($object, $asType=null) {
 			}
 		}
 
+		// Populate named methods
+		if (isset($this->specs[$objtype]['methods'])) {
```

Comment:
This will error in the case that `$this->specs[$objtype]` isn't set. Use:

`if (isset($this->specs[$objtype]) && isset($this->specs[$objtype]['methods'])) {`

Or alternatively combine with the block above this block.